plugins/compare/frontend/public/javascripts/countly.views.js

```diff
@@ -1,488 +1,165 @@
+                    if (selectedItem === this.i18n("events.compare.results.by.sum")) {
+                        self.selectedMetric = "Sum";
+                        this.$store.dispatch('countlyCompareEvents/fetchSelectedGraphMetric', "s");
+                        this.$store.dispatch('countlyCompareEvents/fetchLineChartData', selectedEvents);
+                    }
+                    else if (selectedItem === this.i18n("events.compare.results.by.duration")) {
+                        self.selectedMetric = "Duration";
+                        this.$store.dispatch('countlyCompareEvents/fetchSelectedGraphMetric', "dur");
+                        this.$store.dispatch('countlyCompareEvents/fetchLineChartData', selectedEvents);
                     }
                     else {
-                        return row.avgduration.total;
+                        self.selectedMetric = "Count";
+                        this.$store.dispatch('countlyCompareEvents/fetchSelectedGraphMetric', "c");
+                        this.$store.dispatch('countlyCompareEvents/fetchLineChartData', selectedEvents);
                     }
-                },
-                "sType": "numeric",
-                "sTitle": jQuery.i18n.map["dashboard.avg-time-spent"]
-            }
-        ];
-    },
-    getAlternatives: function() {
-        var apps = countlyGlobal.apps,
-            ids = _.pluck(apps, "_id"),
-            names = _.pluck(apps, "name");
-
-        var toReturn = {};
-
-        for (var i = 0; i < ids.length; i++) {
-            toReturn[ids[i]] = names[i] || ids[i];
+                }
+            },
+        },
+        data: function() {
+            return {
+                value: "",
+                maxLimit: 20,
+                availableMetrics: [
+                    this.i18n("events.compare.results.by.count"),
+                    this.i18n("events.compare.results.by.sum"),
+                    this.i18n("events.compare.results.by.duration")
+                ],
+                selectedMetric: "Count"
+            };
+        },
+        beforeCreate: function() {
+            countlyCommon.setPeriod("30days");
+            this.$store.dispatch('countlyCompareEvents/fetchAllEventsData');
         }
 
-        return toReturn;
-    },
-    beforeRender: function() {
-        $("body").addClass("compare-apps-view");
-        $("#sidebar").addClass("hidden");
-        $("#app-navigation").removeClass("active");
-        $("#hide-sidebar-button").hide();
-    },
-    onRender: function() {
-        $("#content-container").addClass("cover-left");
-
-        $(".app-navigate").on("click", function() {
-            var appId = $(this).data("id");
+    });
 
-            if (countlyCommon.ACTIVE_APP_ID === appId) {
-                app.navigate("/", true);
-            }
+    if (countlyAuth.validateRead(FEATURE_NAME)) {
+        countlyVue.container.registerTab("/analytics/events", {
+            priority: 2,
+            name: "Compare Events",
```

Comment:
name should be whithout white spaces.
https://docs.google.com/document/d/1LBkmpv-HjBnYcjNf86tHiOWYEWjnxQwxQVRjXGSf5yQ/edit?pli=1#heading=h.amk244vn9egw